services/darksky_service.js

```diff
@@ -0,0 +1,16 @@
+var fetch = require('node-fetch');
```

Comment:
@bplantico What do you think about the implementation here?  Not as pretty as Rails, but it's doing the job.

Reply:
This looks great actually.
Fetch may actually be ✨cleaner ✨ than Faraday, IMO....